src/pattern-library/components/patterns/UsingComponentsPage.js

```diff
@@ -72,16 +76,145 @@ export default function UsingComponentsPage() {
  *   outermost element.
  */
 `}
-        />
+            title="Common presentational-component props"
+          />
+
+          <p>
+            Presentational components also take <strong>HTML attributes</strong>{' '}
+            applicable to their outermost element.
+          </p>
+        </Library.Example>
         <Library.Example title="Composite components">
           <p>
-            Composite components accept the same common props as presentational
-            components with the exception of{' '}
-            <s>
-              <code>classes</code>
-            </s>
-            .
+            <strong>Composite components</strong> accept the same common props
+            as presentational components with the exception of{' '}
+            <code>classes</code>.
+          </p>
+        </Library.Example>
+
+        <Library.Example title="Simple components">
+          <p>
+            <strong>Simple components</strong> take only documented,
+            component-specific props.
+          </p>
+        </Library.Example>
+      </Library.Pattern>
+
+      <Library.Pattern title="Other prop conventions">
+        <Library.Example title="variant">
+          <p>
+            Many components provide stylistic <strong>variants</strong>.
+            Variants are mutually exclusive and may affect several different
+            aspects of styling.
+          </p>
+          <p>
+            In contrast, other props like <code>underline</code> or{' '}
+            <code>size</code> impact a single aspect of styling and may be
+            combined in various ways.
+          </p>
+          <Library.Demo
+            title="Card variants, affecting shadows and hover effects"
+            withSource
+          >
+            <Card variant="raised">
+              <CardContent>
+                This is content inside of a <code>Card</code> with the default,{' '}
+                <code>raised</code> <code>variant</code>.
+              </CardContent>
+            </Card>
```

Comment:
The JSX formatting in the pattern library currently always puts new elements (eg. `<code>` here) on a separate line. It would be good to tweak that in future to be more like Prettier's output.